test/property.test.js

```diff
@@ -62,37 +58,127 @@ describe('loopback:property generator', function() {
     });
   });
 
-  it('creates a typed array', function(done) {
+  it('should create model containing number type', function(done) {
     var propertyGenerator = givenPropertyGenerator();
     helpers.mockPrompt(propertyGenerator, {
       model: 'Car',
-      name: 'list',
+      name: 'age',
+      type: 'number',
+      required: 'true',
+      defaultValue: '555555555555555555555.5'
```

Comment:
Unless there is an intention in the number of digits in this number, then please use the simplest numeric value that works, i.e. `0`, `1` or `1.2`.